src/content/en/2020/performance.md

```diff
@@ -228,58 +240,64 @@ First Contentful Paint (FCP) measures the first time at which the browser render
+{# TODO - font looks wrong in chart. Asked for edit access to the sheet #}
+
+In the charts above, the FCP distributions are broken down by desktop and mobile. [Comparing to last year](../2019/performance#fcp-by-device), there are noticeably less average FCP readings, while the percentage of fast and slow user experiences has risen no matter the device type. We can still observe the same trend, where mobile users will experience slower FCP more frequently than desktop users. Overall, users are more likely to have a good or poor experience, rather than a mediocre experience.
 
 {{ figure_markup(
   image="performance-fcp-mobile-year-on-year.png",
   caption="A comparison of distribution of websites labeled as having good, needs improvement and poor FCP mobile performance between 2019 and 2020.",
   description="Bar chart showing how the distribution of good, needs improvement and poor mobile FCP has changed between 2019 and 2020. In 2020, 75% of websites have a sub-par FCP.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=2037503700&format=interactive",
-  sheets_gid="2077755325",
+  sheets_gid="1270303192",
   sql_file="web_vitals_by_device.sql"
   )
 }}
 
-Comparing FCP on mobile devices on a year-over-year basis, we observe fewer good experiences and more moderate and poor experiences. **75% of websites have sub-par FCP**. We can speculate (TODO: maybe add a link to images chapter or js chapter if there are findings there) this high percentage of less than ideal FCP readings is a source of frustration and degraded user experience.
+Comparing FCP on mobile devices on a year-over-year basis, we observe fewer good experiences and more moderate and poor experiences. 75% of websites have sub-par FCP. We can speculate {# TODO: maybe add a link to images chapter or js chapter if there are findings there #} this high percentage of less than ideal FCP readings is a source of frustration and degraded user experience.
 
-It’s challenging to deduce why FCP isn’t improving, as numerous factors can delay paints, such as server latency (measured by a handful of metrics, such as TTFB (TODO: add TTFB section link) and RTT), blocking JavaScript requests, or inappropriate handling of custom fonts to name a few.
+It is challenging to deduce why FCP isn't improving, as numerous factors can delay paints, such as server latency (measured by a handful of metrics, such as [Time to First Byte (TTFB)](#time-to-first-byte-ttfb) and RTT), blocking JavaScript requests, or inappropriate handling of custom fonts to name a few. {# TODO - are those not good reasons? r do you mean it's challenging to pinpoint exactly which of these is the cause (or even something else). Consider to reword #}
```

Comment:
@thefoxis - thoughts?

Reply:
@bazzadp what I’m saying here is that it's challenging to deduce why FCP is the way it is because paints can be delayed by a multitude of factors (some listed above). I don’t think we have the data (other than slow TTFB) to decidedly say "x is making FCP bad". poor TTFB definitely affects it but it’s highly contextual

Reply:
I've changed to this to just this:

> Numerous factors can delay paints, such as server latency (measured by a handful of metrics, such as [Time to First Byte (TTFB)](#time-to-first-byte-ttfb) and RTT), blocking JavaScript requests, or inappropriate handling of custom fonts to name a few.

Let me know if you'd prefer something else.